### PR TITLE
Steward fix and IncludeHeader option

### PR DESCRIPTION
**Pull request by uaArsen**
#9 #30

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,19 +27,18 @@
   },
   "homepage": "https://github.com/elasticio/csv-component#readme",
   "dependencies": {
+    "axios": "^0.19.0",
```
Comment (ShkarupaNick) [inline package.json]: remove ^ from version of modules

Comment (uaArsen) [reply package.json]: done

File: lib/read.js [hunk 2]
```diff
@@ -123,16 +129,23 @@ async function ProcessRead(msg, cfg) {
...
-            if (response.statusCode !== 200) {
-                await that.emit('error', 'Unexpected response code code=' + response.statusCode);
-                ended = true;
-                throw Error('Unexpected response code code=' + response.statusCode);
-            }
-        })
-        .pipe(parser).pipe(writer);
+    const ax = axios.create();
+    util.addRetryCountInterceptorToAxios(ax);
+    const response = await ax({
```
Comment (ElBabbler) [inline lib/read.js]: Does Axios use IO stream to make PUT here?

Comment (uaArsen) [reply lib/read.js]: It is using streams. responseType: 'stream',

File: .eslintrc.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const NEVER = 'never';
 
 module.exports = {
     'parserOptions': {
-        ecmaVersion: 2017
+        ecmaVersion: 2019
```
Comment (ElBabbler) [inline .eslintrc.js]: Is it possible to go to airbnb here?

Comment (uaArsen) [reply .eslintrc.js]: We have task to rewrite this component, from scratch in such task yes, in this task no.

File: lib/read.js [hunk 4]
```diff
@@ -123,16 +129,23 @@ async function ProcessRead(msg, cfg) {
...
+    debug('Have got response status=%s headers=%j', response.status, response.headers);
+    if (response.status !== 200) {
+        await that.emit('error', 'Unexpected response code code=' + response.status);
+        ended = true;
+        throw Error('Unexpected response code code=' + response.status);
+    }
+    response.data.pipe(parser).pipe(writer);
 
     // Need to wait for processing all messages
     while (!ended) {
```
Comment (ElBabbler) [inline lib/read.js]: To avoid this cycle it's possible to wrap lines 90-135 with Promise

**Pull request merged**